src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule72thesummaryfragment/InputIncorrectSummaryJavaDocCheck.java

```diff
@@ -85,32 +85,32 @@ void foo6() {}
       */
     InnerInputCorrectJavaDocParagraphCheck anon = new InnerInputCorrectJavaDocParagraphCheck() {
 
-        /**
-         * Some Javadoc.
-         */
-        public static final byte NUL = 0;
+            /**
```

Comment:
you did indentation of comment to the right, it is not required in your issue, please do not do this.
We expect now change in this lines.